tests/shared/audit/30-ospp-v42-4-delete-success.rules

```diff
@@ -1,3 +1,3 @@
 ## Successful file delete
--a always,exit -F arch=b32 -S unlink,unlinkat,rename,renameat -F success=1 -F auid>=1000 -F auid!=unset -F key=successful-delete
```

Comment:
Please remove modifications to this file, it is used for testing of a different rule and it does not influence the rule modified by this PR.

Reply:
Done